<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1130</width>
    <height>863</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>while True: note()</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1111</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="topbar">
     <item>
      <widget class="QPushButton" name="new_note_btn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>New Note</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="txt_btn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Get note from .txt</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="remove_this_note_btn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Remove this note</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="remove_multiple_notes_btn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Remove multiple notes</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>239</x>
      <y>59</y>
      <width>881</width>
      <height>751</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="mainwindow">
     <item>
      <widget class="QGroupBox" name="workarea">
       <property name="title">
        <string>Your Note</string>
       </property>
       <widget class="QPlainTextEdit" name="new_note_text">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>120</y>
          <width>691</width>
          <height>551</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="save_note_as_btn">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>680</y>
          <width>221</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>Save the Note as</string>
        </property>
       </widget>
       <widget class="QLabel" name="note_name_title">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>40</y>
          <width>531</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Note 1</string>
        </property>
       </widget>
       <widget class="QPushButton" name="convert_to_txt_btn">
        <property name="geometry">
         <rect>
          <x>470</x>
          <y>680</y>
          <width>221</width>
          <height>71</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Convert to .txt</string>
        </property>
       </widget>
       <widget class="QPushButton" name="save_note_btn">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>680</y>
          <width>231</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>Save the Note</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="info_about_note">
        <property name="geometry">
         <rect>
          <x>700</x>
          <y>210</y>
          <width>161</width>
          <height>381</height>
         </rect>
        </property>
        <property name="title">
         <string>Information</string>
        </property>
        <widget class="QLabel" name="lines_number_value">
         <property name="geometry">
          <rect>
           <x>8</x>
           <y>110</y>
           <width>141</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>0 lines</string>
         </property>
        </widget>
        <widget class="QLabel" name="lines_number">
         <property name="geometry">
          <rect>
           <x>8</x>
           <y>70</y>
           <width>141</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>Lines number:</string>
         </property>
        </widget>
        <widget class="QPushButton" name="update_info_button">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>320</y>
           <width>161</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>Update info</string>
         </property>
        </widget>
       </widget>
       <widget class="QSpinBox" name="all_size_spin">
        <property name="geometry">
         <rect>
          <x>801</x>
          <y>30</y>
          <width>61</width>
          <height>21</height>
         </rect>
        </property>
        <property name="accessibleDescription">
         <string/>
        </property>
        <property name="minimum">
         <number>12</number>
        </property>
        <property name="maximum">
         <number>22</number>
        </property>
        <property name="value">
         <number>17</number>
        </property>
       </widget>
       <widget class="QLabel" name="all_font_size_label">
        <property name="geometry">
         <rect>
          <x>610</x>
          <y>30</y>
          <width>181</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>All text Font size</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="note_size_spin">
        <property name="geometry">
         <rect>
          <x>801</x>
          <y>70</y>
          <width>61</width>
          <height>21</height>
         </rect>
        </property>
        <property name="minimum">
         <number>12</number>
        </property>
        <property name="maximum">
         <number>30</number>
        </property>
        <property name="value">
         <number>17</number>
        </property>
       </widget>
       <widget class="QLabel" name="note_font_size_label">
        <property name="geometry">
         <rect>
          <x>610</x>
          <y>70</y>
          <width>181</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Note text Font size</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QScrollArea" name="scroll">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>138</y>
      <width>239</width>
      <height>671</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>237</width>
       <height>669</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="number_of_notes_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>You have 1 note</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1130</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
